metrics.yaml

```diff
@@ -175,6 +175,27 @@ launch_event:
       source:
         description: Where the browser was launched from
         type: string
+  failed_browser_launch:
+    type: event
+    lifetime: ping
+    description: |
+      The browser did not launch due to a native messaging host fault. (Chromium Only)
+    bugs:
+      - https://linktobugs.page
+    data_reviews:
+      - https://linktodatareviews.page
```

Comment:
We should put something good in these links.
For the bugs, can you link to the main bug for this? Did we log one yet? All of the entries should link to the main tracking bug.

Reply:
I think the GitHub issue for data review would be the link here. For the others, I'm not sure what to put. Should probably figure them all out before changing anything

Reply:
Can we just use that link for all of them?
If you don't think that's a good fit, then we can check with Amir for a link to whatever doc should be linked to for this.

Reply:
Resolving for future patch: determining all glean fields